content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -41,68 +42,66 @@ for FILE in "${FILES[@]}"; do
 done
 ```
 
-Then verify the blob by using `cosign`:
+Then verify the blob by using `cosign verify-blob`:
 
 ```shell
-cosign verify-blob "$BINARY" --signature "$BINARY".sig --certificate "$BINARY".cert
+cosign verify-blob "$BINARY" --signature "$BINARY".sig --certificate "$BINARY".cert --certificate-identity krel-staging@k8s-releng-prod.iam.gserviceaccount.com --certificate-oidc-issuer https://accounts.google.com
```

Comment:
Does this auto-wrap or should we wrap it manually?

Reply:
I'd prefer a manual wrapping for this command. It is too long to fit on the screen if we don't wrap it. The command is hard to parse (in brain) if not wrapped. Horizontal scrolling works, but ... it is still hard to get an idea of the whole command.

```suggestion
cosign verify-blob "$BINARY" \
  --signature "$BINARY".sig \
  --certificate "$BINARY".cert \
  --certificate-identity krel-staging@k8s-releng-prod.iam.gserviceaccount.com \
  --certificate-oidc-issuer https://accounts.google.com
```

Reply:
I'd prefer a manual wrapping for this command. It is too long to fit on the screen if we don't wrap it. The command is hard to parse (in brain) if not wrapped. Horizontal scrolling works, but ... it is still hard to get an idea of the whole command.

```suggestion
cosign verify-blob "$BINARY" \
  --signature "$BINARY".sig \
  --certificate "$BINARY".cert \
  --certificate-identity krel-staging@k8s-releng-prod.iam.gserviceaccount.com \
  --certificate-oidc-issuer https://accounts.google.com
```